CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ before starting to add changes. Use example [placed in the end of the page](#exa
 - [#73](https://github.com/OS2Forms/os2forms/pull/73a)
   Fix issue with nested elements in webform inherit
 - [#72](https://github.com/OS2Forms/os2forms/pull/72)
-  Fix certificate testing, also testing for RSA/PEM certs as well as PKCS12
+  Closed this pull request, and undid changes in develop
```

Comment:
Since this is a clean up (revert of changes) we should not mention it in the change log, i.e. remove this line plus the line above it.